bedrock/firefox/urls.py

```diff
@@ -64,7 +64,7 @@
                                       template_context_variations=['a']),
         name='firefox.features.private-browsing'),
     url(r'^firefox/ios/testflight/$', views.ios_testflight, name='firefox.ios.testflight'),
-    page('firefox/mobile', 'firefox/mobile.html'),
+    url(r'^firefox/mobile', views.firefox_mobile, name='firefox.mobile'),
```

Comment:
I took a quick peek and the failing integration tests seem to be redirects. I think changing this to `r'^firefox/mobile/$'` should fix the issue.

Reply:
Thanks @alexgibson!